pwa/src/templates/templateParts/resultsTemplates/cards/CardsResultTemplate.tsx

```diff
@@ -21,8 +21,9 @@ export const CardsResultTemplate: React.FC<CardsResultTemplateProps> = ({ compon
           labelsWithIcon={[
             { label: "Laag", icon: <HamburgerIcon /> },
             { label: "Organisatie", icon: <HouseIcon /> },
+            { label: "Installations", icon: <SettingsIcon /> },
           ]}
-          tags={[component.developmentStatus, component.softwareType]}
+          tags={[component.developmentStatus, component.softwareType , component.usedBy.length]}
```

Comment:
`usedBy` can be `null`.

In the used test data set, `usedBy` always contains an array. Therefore, this works now. However when we do get other data and it is set to `null`, this would break.

Because: `null.length` doesn't exist. 

To fix this: add an optional operator, like so: `component.usedBy?.length`.

Also, once we no longer can get an error, we still want to make sure the frontend receives a value to show (if `null` the value to the tag should be `0`).

Maybe a final solution could be:

```
component.usedBy ? component.usedBy.length : 0
```